pkg/templates/charts/toggle/grc/templates/grc-policy-propagator-webhook-service.yaml

```
@@ -11,4 +11,4 @@ spec:
     protocol: TCP
     targetPort: 9443
   selector:
-    webhook-origin: governance-policy-propagator
\ No newline at end of file
+    name: governance-policy-propagator
```

I think matching a label for the pod label selector would be safer.
```suggestion
    component: ocm-policy-propagator
```

---

oh metrics has ` component: ocm-policy-propagator`